_rules/SC3-1-1-html-xml-lang-match.md

```diff
@@ -0,0 +1,81 @@
+---
+name: SC3-1-1-html-xml-lang-match
+
+description: |
```

Comment:
In the "view" mode it seems that link insertion is not possible in the description.

Reply:
Yeah, I don't think GH view supports it. I had to do some magic to make markup in the description work in the website. It should work there.